src/Vlc.DotNet.Core.Interops/Signatures/libvlc_media_player.h/libvlc_get_fullscreen.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Vlc.DotNet.Core.Interops.Signatures
+{
+    /// <summary> 
+    /// Set fullscreen. 
+    /// </summary>
+    [LibVlcFunction("libvlc_get_fullscreen")]
+    [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+    internal delegate bool GetFullScreen(IntPtr mediaPlayerInstance);
```

Comment:
I'd prefer to return an `int` there, because the function is declared to use a int on vlc side. I don't know what wrong marshalling can do.

Reply:
Sure, I can do that. However, I tested the project by using it's dll in my project and the getter worked just fine.

Reply:
okay will change it to int. You are welcome. It was a need as your project is awesome and I wanted to use the stable and updated version of it.

Reply:
Does the function return a negative int?

Reply:
I don't think so, but comparing strictly to 0 seems like a good general approach for dealing with boolean values (they might add a semantic in the future, like 1 for screen 1, 2 for screen 2...)